### ISSUE TITLE
ERROR - Failed to load the extension "coqui_tts" on Linux

### ISSUE THREAD

**Issue by Galaxia-mk** [oobabooga/text-generation-webui#5360]
### Describe the bug

I can't load coqui_tts anymore. It gives me errors when I try.

Even after running these it won't work
"pip install -r extensions/coqui_tts/requirements.txt --upgrade
pip install --upgrade tts"

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

I ran the update_linux instead of start_linux once and after that it didn't want to load the extension anymore. 

### Screenshot

![pandas](https://github.com/oobabooga/text-generation-webui/assets/110353717/4664f550-08fb-4f86-8c1f-301da7290eda)
![ui error](https://github.com/oobabooga/text-generation-webui/assets/110353717/db77cf5e-858e-4db1-9f15-60879e62a45e)

### Logs

```shell
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
tts 0.22.0 requires pandas<2.0,>=1.4, but you have pandas 2.2.0 which is incompatible.




19:50:10-099577 ERROR    Failed to load the extension "coqui_tts".                                            
Traceback (most recent call last):
  File "/home/galaxia/text-generation-webui-main/modules/extensions.py", line 47, in load_extensions
    extension.setup()
  File "/home/galaxia/text-generation-webui-main/extensions/coqui_tts/script.py", line 168, in setup
    model = load_model()
            ^^^^^^^^^^^^
  File "/home/galaxia/text-generation-webui-main/extensions/coqui_tts/script.py", line 64, in load_model
    model = TTS(params["model_name"]).to(params["device"])
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/api.py", line 74, in __init__
    self.load_tts_model_by_name(model_name, gpu)
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/api.py", line 177, in load_tts_model_by_name
    self.synthesizer = Synthesizer(
                       ^^^^^^^^^^^^
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/utils/synthesizer.py", line 109, in __init__
    self._load_tts_from_dir(model_dir, use_cuda)
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/utils/synthesizer.py", line 161, in _load_tts_from_dir
    config = load_config(os.path.join(model_dir, "config.json"))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/config/__init__.py", line 97, in load_config
    config_class = register_config(model_name.lower())
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/config/__init__.py", line 40, in register_config
    from TTS.tts.configs.xtts_config import XttsConfig
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/tts/configs/xtts_config.py", line 5, in <module>
    from TTS.tts.models.xtts import XttsArgs, XttsAudioConfig
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/tts/models/xtts.py", line 12, in <module>
    from TTS.tts.layers.xtts.stream_generator import init_stream_support
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/tts/layers/xtts/stream_generator.py", line 24, in <module>
    from transformers.generation.utils import GenerateOutput, SampleOutput, logger
ImportError: cannot import name 'SampleOutput' from 'transformers.generation.utils' (/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/transformers/generation/utils.py)
```

### System Info

```shell
Operating system: Kubuntu 22.04
GPU brand: Nvidia
GPU model: A5000
```

**Comment by hydroculator** [comment]
Getting the same thing on Windows. It looks like the update_windows.bat (and the screenshot above from Linux) uninstalls Panda 1.5, installs Panda 2,2, then complains that Panda is greater than 2. A version mismatch in a req.txt file somewhere?

**Comment by Galaxia-mk** [comment]
> Getting the same thing on Windows. It looks like the update_windows.bat (and the screenshot above from Linux) uninstalls Panda 1.5, installs Panda 2,2, then complains that Panda is greater than 2. A version mismatch in a req.txt file somewhere?

I was thinking of that possibility too so I tried to reinstall it but I didn't succeed. In the end I just came to the conclusion that non of my hardware or my software was the problem. It's very likely the web ui package so now I'm just hoping for ooba to fix the issue by probably having the web ui install pandas 1.5 instead of 2.2 again. Either that or Coqui needs to upgrade to 2.2 and we would just need to wait for that instead. 
For now I'll just use silero tts or nothing.

**Comment by Ph0rk0z** [comment]
you can install with nodeps like pip install TTS --no-deps Sometimes there is a hard dependency conflict but often there is not.

**Comment by Galaxia-mk** [comment]
> you can install with nodeps like pip install TTS --no-deps Sometimes there is a hard dependency conflict but often there is not.

I tried it but even though it appeared to install TTS. When i ran it it gave me this 

"ERROR    Failed to load the extension "coqui_tts".                              
Traceback (most recent call last):
  File "/home/galaxia/text-generation-webui-main/modules/extensions.py", line 36, in load_extensions
    exec(f"import extensions.{name}.script")
  File "<string>", line 1, in <module>
  File "/home/galaxia/text-generation-webui-main/extensions/coqui_tts/script.py", line 9, in <module>
    from TTS.api import TTS
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/api.py", line 9, in <module>
    from TTS.utils.audio.numpy_transforms import save_wav
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/utils/audio/__init__.py", line 1, in <module>
    from TTS.utils.audio.processor import AudioProcessor
  File "/home/galaxia/text-generation-webui-main/installer_files/env/lib/python3.11/site-packages/TTS/utils/audio/processor.py", line 4, in <module>
    import librosa
ModuleNotFoundError: No module named 'librosa' "


I remember that in the past coqui functioned without the need for the command pip install TTS. I only needed 
"pip install -r extensions/coqui_tts/requirements.txt --upgrade"

**Comment by Ph0rk0z** [comment]
did you install librosa?

**Comment by Galaxia-mk** [comment]
> did you install librosa?

I installed it now. There were a lot of modules it wanted me to install. After I installed all of them it did seem to get further because it started to download the model. But now that it's done i see this:
![error](https://github.com/oobabooga/text-generation-webui/assets/110353717/ca7c1ab7-e34d-47c1-ac1f-0e81caf9df65)

**Comment by Ph0rk0z** [comment]
You can try to downgrade transformers to 4.36.2. I haven't tried it in the new one yet. I should see it still works.

**Comment by Galaxia-mk** [comment]
> You can try to downgrade transformers to 4.36.2. I haven't tried it in the new one yet. I should see it still works.

Thanks mate. I got it to work now. Not entirely sure if it was just the transformers command that did it but first i ran:
"pip install --upgrade pandas==1.5"

Then because gruut was the wrong version I used the command:
"pip install --upgrade gruut==2.2.3"

And then i did the transformers command:
"pip install --upgrade transformers==4.36.2"

"--upgrade" was said to be used for downgrading as well. And it works.
![solved](https://github.com/oobabooga/text-generation-webui/assets/110353717/54fe3ba6-3681-4305-8084-842973ef4b91)

I wonder if i should not close this thread in case others have questions and because ooba still should be aware of that his is an issue that got to be solved in the standard web-ui package.
